source/sync/configure/sync-settings.txt

```diff
@@ -191,37 +191,37 @@ determine a user's ability to sync, view, and modify data.
 For details, see :ref:`sync-permissions`.
 
 .. _asymmetric-sync:
-.. _optimize-asymmetric-sync:
```

Comment:
just curious -- we fully delete/rename x-ref targets instead of just adding new ones? (I might be conflating with redirects, which I was told to not delete)